controllers/result_test.go

```diff
@@ -21,6 +21,22 @@ var _ = Describe("ResultController", func() {
 
 				Expect(response.StatusCode).To(Equal(http.StatusOK))
 			})
+
+			Context("given user have results", func() {
+				It("renders user results", func() {
+					user := FabricateUser(faker.Email(), faker.Password())
+					otherUser := FabricateUser(faker.Email(), faker.Password())
+					result1 := FabricateResult(user)
+					result2 := FabricateResult(user)
+					result3 := FabricateResult(otherUser)
+					response := MakeAuthenticatedRequest("GET", "/", nil, nil, user)
```

Comment:
Could we add a space before this line so that it separates from the creating test data section?

Reply:
✅  updated in [466be1a](https://github.com/carryall/go-google-scraper-challenge/pull/65/commits/466be1ac5573f03b4aac96aaa7d391379fb53be0)